src/__stories__/config.js

```diff
@@ -7,7 +7,10 @@ function loadStories() {
 }
 
 configure(loadStories, module)
-Events.onEvent(data => console.log("Tracked event", data))
+
+Events.onEvent(data => {
+  // console.log("Tracked event", data)
+})
```

Comment:
I think we wanna toggle this log back on?

Reply:
Ahh yes. Would like to add an ENV var so as to not clutter up our dev environment but wil switch on for now